src/main/scala/org/constellation/ConstellationNode.scala

```diff
@@ -81,14 +82,28 @@ object ConstellationNode extends IOApp {
       fromArg.getOrElse(Option(System.getenv(env)).map(_.toInt).getOrElse(config.getInt(configPath)))
     }
 
+  private def getKeyPair[F[_]: Sync](cliConfig: CliConfig): F[KeyPair] =
+    if (cliConfig.keyStorePath != null) {
+      KeyStoreUtils
+        .fromPath(
+          cliConfig.keyStorePath,
```

Comment:
Try to use application.conf instead so the parsing, validation is done automatically values can be pass as java properties e.g  `java -Dconstellation.keystore.path`

Reply:
I thought that`application.conf` is completely fine for keeping constant things there but for things provided externally (keyStorePath could vary between the users and there is no any default path) we use `ClionfigParser.scala`. Or maybe I missed something 🤔 .

I am wondering why we keep some things in `application.conf` and some things in `CliConfig`, is there any pattern behind that?

Reply:
CliConfig is obsolete it was here we are trying to slowly get rid of it, typesafe config advantage is that you can have multiple layers of config (overriding  by env variables command lines etc.) it's not strictly couples to file only but also to command line arguments environmental variables you can check the examples https://github.com/lightbend/config

Reply:
Ok, good to know :) Maybe let's make the transition in next PR as I need to combine it with terraform config anyway and that one stands for working MVP.

Reply:
Ok, I will do that in the next PR when preparing the terraform/gcp setup to leave that PR atomic.